lib/runtime/life/resolver.go

```diff
@@ -134,7 +134,7 @@ func (*Resolver) ResolveGlobal(_, _ string) int64 {
 }
 
 func ext_logging_log_version_1(vm *exec.VirtualMachine) int64 {
-	logger.Trace("[ext_logging_log_version_1] executing...")
+	logger.Trace("executing...")
```

Comment:
I have always wondered what `...` is for?

Reply:
...For suspense... :thinking: